packages/internal-input-option/src/InputOption/InputOption.style.ts

```diff
@@ -0,0 +1,131 @@
+import { css } from '@leafygreen-ui/emotion';
+import { Theme } from '@leafygreen-ui/lib';
+import { palette } from '@leafygreen-ui/palette';
+import {
+  fontFamilies,
+  spacing,
+  transitionDuration,
+  typeScales,
+} from '@leafygreen-ui/tokens';
+
+export const inputOptionStyles = css`
+  position: relative;
+  list-style: none;
+  outline: none;
+  border: unset;
+  margin: 0;
+  text-align: left;
+  text-decoration: none;
+
```

Comment:
maybe we can add a section about grouping CSS in the style guide?

Reply:
Yeah, I like that. This is my preferred one https://github.com/ream88/stylelint-config-idiomatic-order